private/src/styles/blocks/core-blocks/_navigation.scss

```diff
@@ -0,0 +1,67 @@
+.wp-block-navigation {
+  padding: 20px;
```

Comment:
Padding should perhaps be set on `.wp-block-navigation__container`, as that is the scroll container, and it looks a little better when scroll bars are visible.

Reply:
resolved